packages/gestalt/src/Avatar/Avatar.js

```diff
@@ -50,6 +50,10 @@ const DefaultAvatar = ({ name }: { name: string }) => {
   );
 };
 
+type State = {
```

Comment:
Can this be an exact object? `{| isImageLoaded: boolean |}` and come below the `Props` type please?